v3/servers.go

```diff
@@ -0,0 +1,191 @@
+// Package v3 provides primitives to interact with the openapi HTTP API.
+//
+// Code generated by github.com/egoscale/v3/generator version v0.0.1 DO NOT EDIT.
+package v3
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"runtime"
+	"strings"
+	"time"
+
+	api "github.com/exoscale/egoscale/v3/api"
+	"github.com/exoscale/egoscale/version"
+)
+
+type RequestMiddlewareFn func(ctx context.Context, req *http.Request) error
+
+// UserAgent is the "User-Agent" HTTP request header added to outgoing HTTP requests.
+var UserAgent = fmt.Sprintf("egoscale/%s (%s; %s/%s)",
+	version.Version,
+	runtime.Version(),
+	runtime.GOOS,
+	runtime.GOARCH)
+
+const pollingInterval = 3 * time.Second
+
+type APIZone string
+
+const (
+	APIZoneCHGva2 = "ch-gva-2"
+	APIZoneCHZrh1 = "ch-zrh-1"
```

Comment:
```suggestion
	APIZoneCHDk2 = "ch-dk-2"
```

Reply:
This file is generated, so the openAPI spec is maybe wrong or not up to date, this is weird, let me check :)

Reply:
It will be fixed in the spec, code will be regenerated and fixed